trp_project_scrum/security/ir_rules_collaborators.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="False">
```

Comment:
we want the rules to be permanent on every update , right? I saw it  on the other module and thought it would be better to impose this stuff on every update JIC it was changed. but ok, it would work changing this.